modules/cluster-logging-about-components.adoc

```diff
@@ -22,7 +22,7 @@ The major components of cluster logging are:
 * visualization - This is the UI component you can use to view logs, graphs, charts, and so forth. The current implementation is Kibana.
 
 ifndef::virt-logging[]
-In this document, this document might refer to log store or Elasticsearch, visualization or Kibana, collection or Fluentd, interchangeably, except where noted.
+In this, document might refer to log store or Elasticsearch, visualization or Kibana, collection or Fluentd, interchangeably, except where noted.
```

Comment:
This still doesn't read very well. Suggest changing it to say simply "This document might..."

Reply:
> @rounaksarda thanks! I will accept this for review by the team. In the future, please open a separate PR for each issue.

@vikram-redhat  ok, I will take care next time. Thanks.

Reply:
@bergerhoffer ok, suggested change is made